src/ui/server/controllers/insurance/declarations/modern-slavery/index.test.ts

```diff
@@ -179,11 +197,45 @@ describe('controllers/insurance/declarations/modern-slavery', () => {
         delete res.locals.application;
       });
 
-      it(`should redirect to ${PROBLEM_WITH_SERVICE}`, () => {
-        post(req, res);
+      it(`should redirect to ${PROBLEM_WITH_SERVICE}`, async () => {
+        await post(req, res);
 
         expect(res.redirect).toHaveBeenCalledWith(PROBLEM_WITH_SERVICE);
       });
     });
+
+    describe('api error handling', () => {
+      describe('save data call', () => {
+        describe('when the save data API call does not return anything', () => {
```

Comment:
should this say false or not return anything?

Reply:
Updated all instances of this.